src/main/java/com/spotify/github/v3/clients/JwtTokenIssuer.java

```diff
@@ -36,9 +37,13 @@
 public class JwtTokenIssuer {
 
   private static final SignatureAlgorithm SIGNATURE_ALGORITHM = SignatureAlgorithm.RS256;
+  private static final long TOKEN_TTL = 600000;
 
   private final PrivateKey signingKey;
-  private static final long TOKEN_TTL = 600000;
+
+  private JwtTokenIssuer(final PrivateKey signingKey) {
```

Comment:
Technically this is breaking API backwards compatibility. I assumed that is fine because this is a helper class and would not be used by consumers of the library, but if we want to keep a public constructor with the same signature we could make that work.